### PR TITLE
linux-persistent-log: add instructions to persist logging

### PR DESCRIPTION
**Pull request by ricardosalveti**
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 8a7dff115d2d414fffadf4625af6337626e26780 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/354/docs/artifacts/html/index.html

File: source/reference/linux-persistent-log.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  CORE_IMAGE_BASE_INSTALL += " \
+      kernel-modules \
+      networkmanager-nmtui \
+      git \
+      vim \
+      packagegroup-core-full-cmdline-utils \
+      packagegroup-core-full-cmdline-extended \
+      packagegroup-core-full-cmdline-multiuser \
+      systemd-journald-persistent \
+  "
```
Comment (EmbeddedAndroid) [inline source/reference/linux-persistent-log.rst]: Should we include the entire image defintion here just to add a package? We have this in the docs already about adding a package: https://docs.foundries.io/latest/customer-factory/extending.html maybe something along those lines.

**Comment by EmbeddedAndroid** [comment]
One minor nit, otherwise looks great. Thanks @rsalveti

File: source/reference/linux-persistent-log.rst (hunk 1 above)
Comment (ricardosalveti) [reply source/reference/linux-persistent-log.rst]: Indeed, forgot we had that already, will link to that instead.

Comment (ricardosalveti) [reply source/reference/linux-persistent-log.rst]: Updated to link to extending instead.

**Comment by doanac** [comment]
Docs for ac620cfe43fdb19c64ee4890049808ae292b147d are browesable at: https://ci.foundries.io/projects/fio-docs/builds/355/docs/artifacts/html/index.html

File: source/reference/linux-persistent-log.rst (hunk 1 above)
Comment (EmbeddedAndroid) [reply source/reference/linux-persistent-log.rst]: lgtm

**Pull request merged**